packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -818,6 +819,16 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 	return <DefaultLineItemSublabel product={ product } />;
 }
 
+function GetBillingIntervalLabel( { product }: { product: ResponseCartProduct } ) {
+	const translate = useTranslate();
+	if ( product.volume > 1 ) {
```

Comment:
This is more of a question as I'm not too familiar with this, but are there ever instances where volume is used for other things than years?

Reply:
I believe `volume` is used for 'renewal intervals' whereas if you were quantifying some other sort of unit like 'license seats' or 'number of x' you would use `quantity`.

Reply:
I looked a little closer and it _appears_ that some products may use `volume` to count the number of copies of the product being sold (similar to how we now use `quantity`) - fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Snqzva%2Qcyhtvaf%2Sjcpbz%2Qovyyvat%2Spynff.jcpbz%2Qfgber.cuc%3Se%3Q351o4024%234807%2Q4823-og

Maybe @michaeldcain knows more here.